Dockerfile

```
@@ -12,7 +12,7 @@ LABEL "com.github.actions.color"="gray-dark"
 # Install git (required by ansible-lint)
 RUN set -ex && apt-get update && apt-get -q install -y -V git && rm -rf /var/lib/apt/lists/*
 
-RUN pip install ansible-lint
+RUN pip install 'ansible-lint==4.3.7'
```

It's probably worth doing
```suggestion
RUN pip install 'ansible-lint<5'
```
for now